app/pages/onboarding/08-set-password/set-password.tsx

```diff
@@ -38,7 +41,11 @@ export const SetPassword: React.FC = () => {
     e.preventDefault();
     if (password === null) return;
     setHasSubmitted(true);
-    setStrengthResult(validatePassword(password));
+    const result = validatePassword(password);
```

Comment:
I know the designs don't include this, but I feel like we should:

- Tell the user the password requirements up front
- Show all errors that the user has - not just one at a time

Reply:
Totally agree, but two points: 
- Would rather designs for this first
- With [zxcvbn](https://github.com/dropbox/zxcvbn) there aren't hard requirements (besides 12char min I added). As such, you get a score. So these tips should be advice on strong passwords, rather than explicit _must have two numbers and a symbol_ etc